MAUI/DataGrid/Grouping.md

```diff
@@ -485,4 +485,101 @@ The visibility of the grouped column can be customized by the [SfDataGrid.ShowCo
 this.dataGrid.ShowColumnWhenGrouped = false;
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Customize Group Icon with template
+
+By default , the SfDatagrid loads two different paths for denoting the group expanded and collapsed state. The group icon can be Customized by the [sfDataGrid.GroupExpandCollapseTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_GroupExpandCollapseTemplate) property. When loading as a template, you have the ability to load a single template that indicates the expanded state of a group. This template will then rotate to represent the collapsed state. Refer to the following code snippet for using template:
```

Comment:
The SfDataGrid uses an icon to indicate the expand and collapse state of groups. You can personalize the group icon by using the [sfDataGrid.GroupExpandCollapseTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_GroupExpandCollapseTemplate) property. This property allows you to define a custom template that will be displayed in its normal form when the group is expanded, and it will rotate downwards when the group is collapsed. To implement this, refer to the following code snippet:

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/05c75fc3-8245-43ae-ad31-e82a5d5604f0)